frontend/src/app/analytics/Analytics.test.tsx

```diff
@@ -409,7 +409,7 @@ describe("Analytics", () => {
     expect(await screen.findByText("447")).toBeInTheDocument();
     expect(await screen.findByText("39.3%")).toBeInTheDocument();
   });
-  it("filters out selection of disabled diseases", async () => {
+  it("filters out HIV form dropdown", async () => {
```

Comment:
I had to separate out the testing against both the flags in the same it block as jest was throwing a fit but I have smoke tested in local dev and the filtering logic is consistent with how both of the flags are initialized in the application.yaml file